ui-chat/.gitignore

```diff
@@ -0,0 +1,93 @@
+# Xcode
```

Comment:
It may be a good idea to have 1 gitignore file (instead of separate ones for chat and call)